src/__test__/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprCompute.test.jsx

```diff
@@ -149,6 +170,25 @@ describe('DiffExprCompute', () => {
   });
 
   it('clicking on the second comparison option changes items in form', () => {
+    const store = mockStore({
+      ...initialState,
+      differentialExpression: {
+        ...initialState.differentialExpression,
+        comparison: {
+          ...initialState.differentialExpression.comparison,
+          type: 'within',
+          group: {
+            ...initialState.differentialExpression.comparison,
+            within: {
+              cellSet: null,
+              compareWith: null,
+              basis: null,
+            },
+          },
+        },
+      },
+    });
+
     const component = mount(
       <Provider store={store}>
         <DiffExprCompute
```

Comment:
do we still need to pass in `DiffExprCompute` `cellSets` as props? Since they are in the redux store, is there any reason why we can't read them directly from the `differentialExpression` store directly inside the `DiffExprCompute` component?

Reply:
Nope, deleted in new commit